doc: relnotes: Add a note about the new home for NCS (#2490)
nrfconnect/sdk-nrf: doc/release-notes-1.3.0.rst

Highlights
**********

* New home for the |NCS|
* Support for Thread and Zigbee
* Support for Bluetooth Mesh
* nRF Desktop supported for production
* New nRF9160 samples and libraries
* Support for using external flash for the secondary slot in MCUboot
* General updates and bugfixes

A new home
**********

> Starting in this release, the |NCS| has moved to a new GitHub organization, `nrfconnect <https://github.com/nrfconnect>`_, and the repositories have all been renamed in order to provide clear an concise names that better reflect the composition of the codebase. If you are a new |NCS| user then there are no additional actions you need to take, but if you had used previous releases of the |NCS| we recommend reading :ref:`this section <repo_move>`.

Reviewer: ```suggestion
Starting with this release, the |NCS| has moved to a new GitHub organization, `nrfconnect <https://github.com/nrfconnect>`_.
All repositories were renamed to provide clear and concise names that better reflect the composition of the codebase. 
If you are a new |NCS| user, there are no actions you need to take.
If you used previous releases of the |NCS|, you should follow the instructions in :ref:`repo_move`.
```
